test/jest/loginCallback.test.tsx

```diff
@@ -70,6 +70,8 @@ describe('<LoginCallback />', () => {
       </Security>
     );
     expect(oktaAuth.handleLoginRedirect).toHaveBeenCalledTimes(1);
+    await Promise.resolve();
```

Comment:
What is the purpose of this `Promise.resolve`?

Reply:
To wait for `handleLoginRedirect` which is a promise

Reply:
You can put the await in the expect call
```javascript
expect(await oktaAuth.handleLoginRedirect)
```
or
```javascript
await expect(oktaAuth.handleLoginRedirect).resolves.toHaveBeenCalled()
```

Reply:
```js
    await act(async () => {
      mount(
        <Security {...mockProps}>
          <LoginCallback />
        </Security>
      );
    });
```
this works without need of `await Promise.resolve();`